sdk/python/kfp/dsl/_container_op.py

```diff
@@ -288,5 +288,15 @@ def add_pod_label(self, name: str, value: str):
     self.pod_labels[name] = value
     return self
 
+  def set_retry(self, num_retry: int):
```

Comment:
BTW, the reason Argo has `retryStrategy.limit` property instead of just `numRetries` is probably because they want to add more ways to do the retries (e.g. exponential backoff). If that feature is added, what's our plan on supporting it?

Reply:
We can decide once argo has more. For example, we can add set_retry_strategy or even deprecate the simple retry. But usually I am in favor of simplicity, especially we don't know argo's plan.